attestation-service/src/main/kotlin/AttestationService.kt

```diff
@@ -105,14 +201,66 @@ data class IOSAttestationConfiguration @JvmOverloads constructor(
             /**
              * @see AppData.iosVersionOverride
              */
-            fun overrideIosVersion(version: String) = apply { iosVersionOverride = version }
+            fun overrideIosVersion(version: OsVersions) = apply { iosVersionOverride = version }
 
             fun build() = AppData(teamIdentifier, bundleIdentifier, sandbox, iosVersionOverride)
         }
     }
 
 }
 
+typealias ParsedVersions = Pair<SemVer?, BuildNumber?>
+
+/**
+ * iOS build number. As per [TidBITS.com](https://tidbits.com/2020/07/08/how-to-decode-apple-version-and-build-numbers/):
```

Comment:
so that comment is duplicated ... is there one place to put this, i.e. so we don't need to update this twice if it changes?

Reply:
removed duplicates, added cross references